src/fidesops/ops/models/privacy_request.py

```diff
@@ -730,6 +731,17 @@ def hash_value(
         return hashed_value
 
 
+class Consent(Base):
```

Comment:
~Do we need the regulation field that's listed in https://github.com/ethyca/fidesops/issues/1160?~ Nope, Neville requested removal above